Marketplace.md

```diff
@@ -0,0 +1,7 @@
+The Market Place is where you'll go in order to sell and buy other items from players. You can get here by clicking Marketplace on the side or clicking on your gold bag above your stockpile. You can see all the various items within the game here that other players are selling. When you click on any item, it'll open a different window that will show the different listings by other players. By default it is listed by price, lowest to highiest. To return, you can click the "Back" button in the top left. 
+<br />
+<b> Selling </b>
+In order to sell items on the market, you can click the "Sell" button also in the top left on the main screen. Here you will see all the items you can sell, as well as items you already have listed on the market if any. When you click on an item, it will bring up a screen asking you to select the amount you wish to sell of that item, and for how much. Once you are satisfied with your listing, click "Create Listing" and your item will now be on the market for other players to purchase with gold should they choose to do so.
```

Comment:
Somethink similar to this
Tip: You can use K,M,B in the price input. E.g. 2.5M for 2,500,000
would be nice.